### ISSUE TITLE
Aria Announce Provider ADA Issue

### ISSUE THREAD

**Issue by jake-costa** [jpmorganchase/salt-ds#5592]
### Latest version

- [x] I have tested the latest version

### Description

Currently the default state for the aria announce provider is assertive so when components use the announcer for announcements that are not assertive, this results in an ADA violation. 

### Steps to reproduce

.

### Expected behavior

Components using aria announce provider need to be able to set it as polite.

### Package name(s)

Core (@salt-ds/core)

### Browser

- [x] Chrome
- [x] Safari
- [x] Firefox
- [ ] Microsoft Edge

### Operating system

- [x] macOS
- [x] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.
